src/components/numberinput/Numberinput.vue

```diff
@@ -193,27 +195,27 @@ export default {
         },
         onControlClick(event, inc) {
             // IE 11 -> filter click event
-            if (event.detail !== 0 || event.type === 'click') return
+            if (event.detail !== 0 || event.type !== 'click') return
             if (inc) this.increment()
             else this.decrement()
         },
+        longPressTick(inc) {
+            if (inc) this.increment()
+            else this.decrement()
+
+            this._$intervalRef = setTimeout(() => {
+                this.longPressTick(inc)
+            }, this.exponential ? (250 / (this.exponential * this.timesPressed++)) : 250)
```

Comment:
Hardcoded, make it configurable (250).

Reply:
The exponential is configurable making the timer speed effectively configurable more than well enough in term of changing speed